invenio_app_ils/circulation/api.py

```diff
@@ -95,13 +95,10 @@ def patron_has_active_loan_or_request_on_document(patron_pid, document_pid):
         patron_pid=patron_pid, document_pid=document_pid, filter_states=states
     )
     search_result = search.execute()
-
-    return (
-        search_result,
-        search_result.hits.total > 0
-        if lt_es7
-        else search_result.hits.total.value > 0,
-    )
+    if len(search_result) > 0:
```

Comment:
I am not 100% sure this is correct because the length should be computed on the `hits` field. WDYT?

```suggestion
    total = search_result.hits.total
        if lt_es7
        else search_result.hits.total.value
    if total > 0:
```

Reply:
Yes, you are right I only tested it in my local environment and didn't consider other ES versions, I will change it, thanks!